controller/src/main/java/org/jboss/as/controller/ObjectListAttributeDefinition.java

```diff
@@ -229,6 +234,19 @@ ObjectTypeAttributeDefinition getValueType() {
         return valueType;
     }
 
+
+    public List<ModelNode> unwrap(final ExpressionResolver context, final ModelNode model) throws OperationFailedException {
+        ModelNode value = resolveModelAttribute(context, model);
+        if (!model.isDefined()) {
+            return Collections.emptyList();
```

Comment:
Why not null? I think a method like this should support null, empty and non-empty. If you don't want the caller to have to deal with null, add an overloaded variant.

This method needs clear javadoc to establish its contract.